central/reports/scheduler/v2/reportgenerator/csv_gen.go

```diff
@@ -38,27 +38,22 @@ func GenerateCSV(cveResponses []*ImageCVEQueryResponse, configName string, inclu
 	}
 	csvWriter := csv.NewGenericWriter(csvHeaderClone, true)
 	for _, r := range cveResponses {
-		discoveredTs := "Not Available"
-		if r.DiscoveredAtImage != nil {
-			discoveredTs = r.DiscoveredAtImage.Format("January 02, 2006")
-		}
 		row := csv.Value{
-			r.Cluster,
-			r.Namespace,
-			r.Deployment,
-			r.Image,
-			r.Component,
-			r.ComponentVersion,
-			r.CVE,
-			strconv.FormatBool(r.Fixable),
-			r.FixedByVersion,
-			strings.ToTitle(stringutils.GetUpTo(r.Severity.String(), "_")),
-			strconv.FormatFloat(r.CVSS, 'f', 2, 64),
-			discoveredTs,
+			r.GetCluster(),
+			r.GetNamespace(),
+			r.GetDeployment(),
+			r.GetImage(),
+			r.GetComponent(),
```

Comment:
Where did `ComponentVersion` go?

Reply:
It is not a part of current VM reports. I added it by mistake when writing this generator. It does make sense to have it though.